src/test/java/io/cucumber/StepDefinitions.java

```diff
@@ -32,7 +34,7 @@
     private final String BASE_URL = "https://www.saucedemo.com";
     private SauceUtils sauceUtils;
 
-    @Before
+    @Before("not @api")
```

Comment:
Why separate API/Not API.  I think the most common use case would be to use the API in conjunction with Selenium -- you sign up, navigate to the download page, and then get the download URL in Selenium, then use the HTTP client to do the actual download.

Reply:
Because if I don't do this then a VM session is created for every single one of my API tests. I don't need a VM for my API test.